.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+
+php:
+  - 7.0
+  - 7.1
+  - 7.2
+  - nightly
+
+matrix:
+  allow_failures:
+    - php: nightly
+
+before_script:
+  - wget https://getcomposer.org/composer.phar
+  - php composer.phar install --no-interaction
+  - composer self-update
+  - git clone https://github.com/octobercms/october.git
+  - mkdir -p ./october/plugins/rainlab/user
+  - mv !(october) october/plugins/rainlab/user
+  - cd october
+  - composer install
+  - cp config/cms.php config/cms.php.testing
+  - mv config/cms.php.testing config/testing/cms.php
```

Comment:
Why don't you just `cp config/cms.php config/testing/cms.php` directly?

Reply:
Ah, good catch, that's a left over from an old project that I copied the config from.